index.bs

```diff
@@ -364,79 +364,80 @@ deletion are considered to be the responsibility of such a user interface and ar
+that are returned to the caller when a new credential is created, or a new assertion is requested.
 
 <xmp class="idl">
     [SecureContext]
-    interface ScopedCredential : Credential {
+    interface PublicKeyCredential : Credential {
         readonly attribute ArrayBuffer           rawId;
         readonly attribute AuthenticatorResponse response;
         readonly attribute AuthenticationExtensions clientExtensionResults;
     };
 </xmp>
-<dl dfn-type="attribute" dfn-for="ScopedCredential">
+<dl dfn-type="attribute" dfn-for="PublicKeyCredential">
     :   {{Credential/id}}
-    ::  This attribute is inherited from {{Credential}}, though {{ScopedCredential}} overrides {{Credential}}'s getter, instead
-        returning the [=base64url encoding=] of the data contained in the object's {{ScopedCredential/[[identifier]]}}
-        [=internal slot=].
+    ::  This attribute is inherited from {{Credential}}, though {{PublicKeyCredential}} overrides {{Credential}}'s getter,
+        instead returning the [=base64url encoding=] of the data contained in the object's
+        {{PublicKeyCredential/[[identifier]]}} [=internal slot=].
 
-    :   {{ScopedCredential/rawId}}
-    ::  This attribute returns the {{ArrayBuffer}} contained in the {{ScopedCredential/[[identifier]]}} internal slot.
+    :   {{PublicKeyCredential/rawId}}
+    ::  This attribute returns the {{ArrayBuffer}} contained in the {{PublicKeyCredential/[[identifier]]}} internal slot.
 
     :   <dfn>response</dfn>
-    ::  This attribute contains the [=authenticator=]'s response to the client's request to either create a [=scoped
-        credential=], or generate an [=authentication assertion=]. If the {{ScopedCredential}} is created in response to
+    ::  This attribute contains the [=authenticator=]'s response to the client's request to either create a [=public key
+        credential=], or generate an [=authentication assertion=]. If the {{PublicKeyCredential}} is created in response to
         {{CredentialsContainer/create()}}, this attribute's value will be an {{AuthenticatorAttestationResponse}}, otherwise,
-        the {{ScopedCredential}} was created in response to {{CredentialsContainer/get()}}, and this attribute's value
+        the {{PublicKeyCredential}} was created in response to {{CredentialsContainer/get()}}, and this attribute's value
         will be an {{AuthenticatorAssertionResponse}}.
 
     :   <dfn>clientExtensionResults</dfn>
     ::  This attribute contains a [=map=] containing [=extension identifier=] → [=client extension output=] entries
         produced by the extension's [=client extension processing=].
 
     :   <dfn>\[[type]]</dfn>
-    ::  The {{ScopedCredential}} [=interface object=]'s {{Credential/[[type]]}} [=internal slot=]'s value is the string
-        "`scoped`".
+    ::  The {{PublicKeyCredential}} [=interface object=]'s {{Credential/[[type]]}} [=internal slot=]'s value is the string
+        "`publickey`".
```

Comment:
yeah, it did seem to me that casing-rules can be interpreted to apply here, but it sorta seems an "internal slot" is only implicitly an enum. If up to me I'd use "publickey" rather than "public-key", but it's not a big deal. this was sort of an experiment to see if anyone would catch it :)